performance-tests/perf-test-commits.sh

```diff
@@ -0,0 +1,113 @@
+#! /usr/bin/env bash
+#
+# Copyright Kroxylicious Authors.
+#
+# Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+#
+
+# perf-test-commits is intended to generate a comparable performance tests results across a series of commits.
+# It will checkout each commit in turn and build an image for each commit before running `pref-test.sh` with each generated image.
```

Comment:
```suggestion
# It will checkout each commit in turn and build an image for each commit before running `perf-tests.sh` with each generated image.
```